components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -87,6 +88,7 @@ struct SiteInfo {
   // Web page specific fields, if available and allowed
   string? title;
   string? hostname;
+  int32? content_id;
```

Comment:
iOS doesn't support nullable int in mojom

Reply:
Well that sucks. I'll update it to use `-1` as no value :cry:

Reply:
Alternatively you could use nullable string or a nullable struct containing the int

Reply:
I think I prefer a negative int for now